kafka-ui-e2e-checks/src/main/java/com/provectus/kafka/ui/pages/connector/ConnectorCreateForm.java

```diff
@@ -3,37 +3,37 @@
 import com.codeborne.selenide.Condition;
 import com.codeborne.selenide.SelenideElement;
 import io.qameta.allure.Step;
-import org.openqa.selenium.By;
 
-import static com.codeborne.selenide.Selenide.$;
+import static com.codeborne.selenide.Selenide.$x;
 import static com.codeborne.selenide.Selenide.sleep;
 import static com.provectus.kafka.ui.utilities.WebUtils.clickByJavaScript;
 import static com.provectus.kafka.ui.utilities.screenshots.Screenshooter.log;
 
 public class ConnectorCreateForm {
 
-    protected SelenideElement nameField = $(By.xpath("//input[@name='name']"));
-    protected SelenideElement contentTextArea = $(".ace_text-input");
-    protected SelenideElement submitButton = $(By.xpath("//button[@type='submit']"));
+    protected SelenideElement loadingSpinner = $x("//*[contains(text(),'Loading')]");
+    protected SelenideElement nameField = $x("//input[@name='name']");
+    protected SelenideElement contentTextArea = $x("//textarea[@class='ace_text-input']");
+    protected SelenideElement submitBtn = $x("//button[@type='submit']");
+    protected SelenideElement configField = $x("//div[@id='config']");
 
-    private static final String path = "/ui/clusters/secondLocal/connectors/create_new";
+    @Step
+    public ConnectorCreateForm waitUntilScreenReady() {
+        loadingSpinner.shouldBe(Condition.disappear);
+        nameField.shouldBe(Condition.visible);
+        return this;
+    }
 
     @Step("Set connector config JSON")
-    public ConnectorDetails setConnectorConfig(String connectName, String configJson) {
-        nameField.setValue(connectName);
-        $("#config").click();
+    public ConnectorCreateForm setConnectorConfig(String connectName, String configJson) {
+        nameField.shouldBe(Condition.enabled).setValue(connectName);
+        configField.shouldBe(Condition.visible).click();
```

Comment:
I believe we can use 'enabled' condition here

Reply:
changed to enabled